### PR TITLE
Feature/keycloak theme

### PR DESCRIPTION
**Pull request by achen2401**
https://www.pivotaltracker.com/story/show/186901516
screenshot of how it will look if works:
Screenshot 2024-01-24 at 12 16 29 PM

### CODE REVIEW

File: dev/config/keycloak/themes/common/resources/lib/zocial/zocial-regular-webfont.eot
Comment (ivan-c) [inline dev/config/keycloak/themes/common/resources/lib/zocial/zocial-regular-webfont.eot:1]: Can you move all the files in `dev/config/keycloak/themes` to `dev/config/keycloak/themes/ltt`? I wasn't able to configure the theme in keycloak until it was in its own (single) directory

Comment (ivan-c) [reply dev/config/keycloak/themes/common/resources/lib/zocial/zocial-regular-webfont.eot:1]: Also, please add a line to the keycloak docker-compose config (see below diff) to mount the theme into the container

```
diff --git a/dev/docker-compose.yaml b/dev/docker-compose.yaml
index f11badb..cc264fd 100644
--- a/dev/docker-compose.yaml
+++ b/dev/docker-compose.yaml
@@ -149,6 +149,7 @@ services:
       - keycloak.env
     volumes:
       - ./config/keycloak/import/:/opt/keycloak/data/import:ro
+      - ./config/keycloak/themes/:/opt/keycloak/themes:ro
     depends_on:
       - db
     networks:
```

**Review by ivan-c — CHANGES_REQUESTED** [review]
Thanks! Had a few suggestions but almost there

File: dev/config/keycloak/themes/common/resources/lib/zocial/zocial-regular-webfont.eot
Comment (ivan-c) [reply dev/config/keycloak/themes/common/resources/lib/zocial/zocial-regular-webfont.eot:1]: Whoops, this change too- to make `ltt` the default `loginTheme` for the `ltt` keycloak realm
```
diff --git a/dev/config/keycloak/import/ltt-realm.json b/dev/config/keycloak/import/ltt-realm.json
index 60c842f..bd9e5c1 100644
--- a/dev/config/keycloak/import/ltt-realm.json
+++ b/dev/config/keycloak/import/ltt-realm.json
@@ -752,6 +752,7 @@
     "user": "${KEYCLOAK_EMAIL_USER}",
     "password": "${KEYCLOAK_EMAIL_PASSWORD}"
   },
+  "loginTheme": "ltt",
   "eventsEnabled": false,
   "eventsListeners": [
     "jboss-logging"
```

**Comment by achen2401** [comment]
@ivan-c  thank you! I believe I addressed all your feedback.  Please let me know if you see any more issue.

**Review by ivan-c — APPROVED** [review]
Looks great, thank you! Please merge at your leisure

**Pull request merged**